src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,60 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px ;
+  padding: 25px;
+
+
+  margin: auto;
+  width: (7 * $cell-width) + 10px;
+  // Use flex with 1px gap
+  // and limit its width to
+  // exactly 7 columns + 10px paddings
+
+  $days:
```

Comment:
better move this map to the variable files

Reply:
DOne!